core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/repair/TestDefaultRepairConfigurationProvider.java

```diff
@@ -101,6 +101,11 @@ public boolean getRemoteRouting()
             {
                 return true;
             }
+
+            @Override
+            public String getSerialConsistency(){
```

Comment:
Incorrect formatting {} should be placed on newlines.

Reply:
Done